app/views/bright_events/user_events.html.erb

```diff
@@ -0,0 +1,66 @@
+<div class="event-list-container">
+  <div>
+    <%= link_to new_bright_event_path , class: 'btn btn-warning' do %>
+      <i class="uil uil-plus"></i>
+      <span>Agregar evento</span>
+    <% end %>
+  </div>
+  <div class="table-container">
+    <table>
+      <tr>
+        <th>Título</th>
+        <th>Descripción</th>
+        <th>Fecha</th>
+        <th>Ubicación</th>
+        <th>Costo</th>
+        <th>Privacidad</th>
+        <th>Imagen</th>
+        <th>Editar</th>
+        <th>Ver</th>
+        <th>Eliminar</th>
+      </tr>
```

Comment:
```suggestion
<tr>
  <% t(  :events, scope: %w[table_headers] ).each do |item| %>
    <th><%=  item %></th>
  <% end %>
</tr>
```

```
en:
  table_headers:
    events:
      title: "Título"
      description: "Descripción"
      date: "Fecha"
      location: "Ubicación"
      cost: "Costo"
      privacy: "Privacidad"
      image: "Imagen"
      edit: "Editar"
      view: "Ver"
      delete: "Eliminar"
```

Reply:
and this same structure can be applied to any textual content on your project, preferably set all the text in the locales

Reply:
Well done!